os/cluster-architectures.md

```diff
@@ -2,43 +2,32 @@
 
 ## Overview
 
-Depending on the size of your cluster and how it's going to be used, there are a few common cluster architectures that can be followed. These are described below.
+Depending on the size and expected use of your Container Linux cluster, you will have different architectural requirements. A few of the common cluster architectures, as well as their strengths and weaknesses, are described below.
 
-Most of these scenarios set aside a few machines dedicated to running central cluster services, such as etcd and the distributed controllers for applications like Kubernetes, Mesos, and OpenStack. Separating out these services into a few known machines allows you to make sure they are distributed across cabinets/availability zones and easily set up static networking to allow for easy bootstrapping. If you're worried about relying on the discovery service, this architecture will remove your worries.
-
-The cloud-config files provided with each section are valid, but you will need to add SSH keys and other desired configuration options.
+Most of these scenarios dedicate a few machines, physical hardware of virtual machines, to running central cluster services. These may include etcd and the distributed controllers for applications like Kubernetes, Mesos, and OpenStack. Excluding these services onto a few known machines helps to ensure they are distributed across cabinets/availability zones. It also helps in setting up static networking to allow for easy bootstrapping. If you're concerned about relying on a discovery service, this architecture helps resolve a lot of those problems.
 
 ## Docker dev environment on laptop
 
 <img class="img-center" src="img/laptop.png" alt="Laptop Environment Diagram"/>
 <div class="caption">Laptop development environment with Container Linux VM</div>
 
-| Cost | Great For | Set Up Time | Production |
-|------|-----------|-------------|------------|
-| Low | Laptop development | Minutes | No |
+| Cost | Great For          | Set Up Time | Production |
+|------|--------------------|-------------|------------|
+| Low  | Laptop development | Minutes     | No         |
 
-If you're developing locally but plan to run containers in production, it helps to mirror that environment locally. This can easily be done by running Docker commands on your laptop that control a Container Linux VM in VMware Fusion or VirtualBox.
+If you're developing locally but plan to run containers in production, it's best practice to mirror that environment locally. This can easily be done by running Docker commands on your laptop that control a Container Linux VM in VMware Fusion or VirtualBox.
```

Comment:
Run Docker commands on your laptop that control a Container Linux VM in VMware Fusion or Virtual box to mirror your container production environment locally.